appveyor.yml

```diff
@@ -1,4 +1,4 @@
-version: 1.0.{build}
+version: 0.9.{build}
```

Comment:
Hmm, we've always used these as cardano version (although it's confusing since it should be daedalus). I'd prefer not to change the convention and go for 1.1